docs/test_py_module/test.py

```diff
@@ -113,6 +114,25 @@ def another_function(self, a, b, **kwargs):
         """
         return sum(kwargs.values()) / len(kwargs), a + b
 
+    @property
+    def qux_caps(self):
+        """Return the instance qux as uppercase.
+
+        :rtype: string
```

Comment:
Can you change these to use ` -> str` annotations instead of `:rtype:` ?

Reply:
done